### DISCUSSION TITLE
Status of Wordpress Plugin?

### DISCUSSION THREAD

**Discussion by sbw** [plausible/analytics#1826] [Anything Else]
Regarding the [WordPress plugin](https://wordpress.org/plugins/plausible-analytics/):

> This plugin has been closed as of April 7, 2022 and is not available for download. This closure is temporary, pending a full review.

What's the status?

I haven't been following development of the plugin.  I remain interested in the proxy (plausible/wordpress#23)?

**Comment by metmarkosaric** [comment]
hi @sbw! 

our plugin is down temporarily due to security improvements and we are working with WP plugins team to resolve this asap. you can find more here: https://github.com/plausible/wordpress/issues/64

we're hoping to have the proxy out of the box in the next major version of the plugin

**Comment by metmarkosaric** [comment]
Our plugin is now back in the WordPress plugin repository. Thank you for your patience!

**Comment by couchfortkid** [comment]
It looks like the Plugin is removed again. Any word on this?

Correction: It was never removed again. I messaged on the wrong forum.

**Comment by sbw** [comment]
@couchfortkid, what makes you say that?

I see it in the Wordpress directory: https://wordpress.org/plugins/plausible-analytics/

**Reply by couchfortkid** [reply]
Sorry I somehow messaged on the wrong forum. It is definitely there!
